module-services/service-evtmgr/battery/BatteryState.cpp

```diff
@@ -136,7 +136,7 @@ class BatteryState::Pimpl
     sml::sm<StateMachine, NotifyStateChangedCallback> sm{notifyCallback};
 
     static constexpr auto criticalThreshold = 10; // %
-    static constexpr auto shutdownThreshold = 5;  // %
+    static constexpr auto shutdownThreshold = 1;  // %
```

Comment:
This shutdown threshold is after scaling?

Reply:
This layer knows nothing about scaling. It only performs a shutdown procedure when battery SOC is lower than 1%. In terms of real SOC values, 1% may translate to 5% or any other value. IMHO it's a much more flexible solution as different hardware may implement this value completely differently. For some systems, it might be reasonable to operate in 10 to 90% and so on.  Currently, both Pure and Bell will be 1% when it is 5%. 

Another thing is battery aging calculations, state of health, etc. It might be also necessary to recalculate these thresholds based on various battery state indicators. It seems like the battery subsystem might be much more complicated than we thought :/ Without a doubt, whole battery management should be removed from the event manager and treated as a separate entity.

Reply:
What do you mean by: `Currently, both Pure and Bell will be 1% when it is 5%.` . For Pure 5% is 5% and we are shutting down it to leave reasonable amount of  residual charge in the battery.

Reply:
Have you had a chance to check this PR entirely? As I said, scaling is performed at hal/BatteryCharger level. Shutdown will be still performed when battery SOC is 5%.

Reply:
As for my question, it has been answered. I believe @wojtekrzepecki concern has also been cleared.